src/pydocstyle/checker.py

```diff
@@ -176,8 +197,24 @@ def checks(self):
             for this_check in vars(type(self)).values()
             if hasattr(this_check, '_check_for')
         ]
+        # This returns the checks in the order they are
+        # listed in the file (since py3.6) if their priority is the same
         return sorted(all, key=lambda this_check: not this_check._terminal)
 
+    # Note - this needs to be listed before other checks
```

Comment:
Checks should not rely on order, other checker functions, etc. Let's find a better solution.
Perhaps my suggestion for returning an empty string from `eval_docstring` is enough to solve this. Otherwise, we can have it return `None` and do more verbose checks at call sites. LMKWYT.